### ISSUE TITLE
"npx avif ..." Broken in node < 12.19 (maybe other versions too)

### ISSUE THREAD

**Issue by Beamanator** [lovell/avif-cli#1]
FYI, I got an error from `require("fs/promises")` - mentioned in [this stack-overflow question](https://stackoverflow.com/questions/64725249/fs-promises-api-in-typescript-not-compiling-in-javascript-correctly) when trying to run `npx avif` with node version 12.13.1

Looks like you may need node > 12.19

Feel free to close this issue if you want, I am just mentioning it b/c you said "requires Node.js 10+" in the Readme :)

**Comment by lovell** [comment]
Thanks for reporting, we should be able to fix this with the following patch. Happy to accept a PR if you're able.

```diff
- const fs = require("fs/promises");
+ const fs = require("fs").promises;
```

**Comment by Beamanator** [comment]
@lovell No problem :) Done 👍

**Comment by lovell** [comment]
Thanks for the PR, published as v0.0.2
